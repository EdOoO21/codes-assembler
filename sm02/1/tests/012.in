-450 346 83 364 -284 -708 -219 -978 -233 -12 -439 317 659 -58 414 770 231 -534 143 -997 354 278 -703 -99 -248 -668 -305 855 -570 -880 180 676 -592 -848 51 406 677 -310 395 -172 634 725 -822 -962 881 -876 678 873 355 42 -543 -814 -133 938 -92 -770 353 -134 989 -724
