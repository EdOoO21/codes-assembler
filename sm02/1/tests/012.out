-724
989
-134
353
-770
-92
938
-133
-814
-543
42
355
873
678
-876
881
-962
-822
725
634
-172
395
-310
677
406
51
-848
-592
676
180
-880
-570
855
-305
-668
-248
-99
-703
278
354
-997
143
-534
231
770
414
-58
659
317
-439
-12
-233
-978
-219
-708
-284
364
83
346
-450
